_posts/2021-06-22-choosing-right-transformation.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Choosing the Right Transformation
+author:
+  display_name: Jake Adams
+  email: jdadams@utah.gov
+date: 2021-06-21 8:30:00
```

Comment:
**nitpick**: let's rename the file to the 23rd and have this date match that as well

Reply:
doh, changed this so it would render in docker and forgot to update it 👍